nginx_proxy/start_nginx.sh

```diff
@@ -79,12 +93,40 @@ if [[ -f "${CONF_FILE}" ]]; then
   cp "${CONF_FILE}" /etc/nginx/nginx.conf
 fi
 
-# fetch Service Configuration from Service Management if the service name and
-# service version are provided.
-if [[ -n "${ENDPOINTS_SERVICE_NAME}" && \
-      -n "${ENDPOINTS_SERVICE_VERSION}" ]]; then
-  /usr/sbin/fetch_service_config.sh \
-    -s "${ENDPOINTS_SERVICE_NAME}" -v "${ENDPOINTS_SERVICE_VERSION}" || exit $?
-fi
+# Start nginx
+if [[ -z "${ENDPOINTS_SERVICE_NAME}" ]]; then
+  /usr/sbin/nginx -p /usr -c /etc/nginx/nginx.conf
+else
+  if [[ "${ENDPOINTS_ROLLOUT_STRATEGY}"  && \
+        "${ENDPOINTS_ROLLOUT_STRATEGY}" != "fixed" && \
+        "${ENDPOINTS_ROLLOUT_STRATEGY}" != "managed" ]]; then
+    echo "Error: rollout strategy option should be either fixed or managed"
+    usage
+  fi
+
+  if [[ "${ENDPOINTS_ROLLOUT_STRATEGY}" != "managed" && \
```

Comment:
managed -> fixed

Reply:
This should be "managed", not "fixed".

Any strategy that is not "managed" is "fixed" and therefore the version must be defined.

Reply:
It checks ENDPOINTS_ROLLOUT_STRATEGY is empty of fixed. If the value is empty, it will be taken as 'fixed'. It should to be 'managed'